docs/rfcs/003-interactive-log-view.md

```diff
@@ -0,0 +1,63 @@
+# Interactive Log View
+
+## Status
+
+Proposed
+
+## Summary
+
+Open a pane item displaying a navigable, live 3D model of your git commit graph.
+
+## Motivation
+
+Visualizing the commit graph is an important tool for learning git. A live view of the commit graph can inobtrusively help git newcomers form the correct mental model for commit graph operations (branching, merging, rebasing).
+
+We do not currently have a way to initiate merge, rebase, or cherry-pick operations from the editor. A log view would be the most logical place to do so, because it's where you have the most context about the existing relationship of the refs you'd like to merge or rebase.
+
+A log view would grant us a way to answer the question of where you are relative to nearby refs and tags.
+
+Why 3D? Flat visualizations of git history quickly fall victim to the "guitar hero problem":
+
+[![Git guitar hero](https://user-images.githubusercontent.com/17565/39223169-6f5d6b1e-480e-11e8-809b-60d1dc8261fb.png)](https://twitter.com/henryhoffman/status/694184106440200192)
+
+I'm hopeful that the extra "space" afforded by rendering in WebGL and level-of-detail fading can give us a view that remains functional even in the face of dramatic tangles.
+
+## Explanation
+
+The log view is a pane item launched by:
+
+* The `github:log` command in the command palette.
+* An "Open Log" menu item in the context menu on the recent commit history.
+
+When launched, the log view pane uses WebGL to render a 3D model of your commit graph.
```

Comment:
@smashwilson Do you have an example of a 3D log visualization or is it something that only exists in your imagination. 💭

Reply:
Totally and completely in my head 😆 

If I get a chance, I might start playing around with a prototype for this to see if it's viable.

Reply:
On a semi-related note, I met some guys who made a git branch viewer for the hololens (it would display the branch graph fixed in space). So I guess that's one realization of a 3D log visualisation